backend/src/api/model/shared.rs

```diff
@@ -243,3 +246,54 @@ where
         page_info,
     })
 }
+
+
+#[derive(Debug)]
+pub(crate) struct AclForDB {
+    // todo: add custom and preview roles for events when sent by frontend
+    pub(crate) read_roles: Vec<String>,
+    pub(crate) write_roles: Vec<String>,
+    // preview_roles: Option<Vec<String>>,
+    // custom_action_roles: Option<CustomActions>,
```

Comment:
When moving this from `event.rs`, you changed the order of fields. The `todo` comment should be above the commented out fields.